applications/MultilevelMonteCarloApplication/external_libraries/XMC/xmc/classDefs_solverWrapper/methodDefs_KratosSolverWrapper/mpi_solve.py

```diff
@@ -38,6 +38,18 @@
 except:
     computing_procs_mlmc_execute_2 = 1
 
+try:
+    ppn_mlmc_execute_0 = int(os.environ["ppn_mlmc_execute_0"])
+except:
+    ppn_mlmc_execute_0 = 1
```

Comment:
```suggestion
ppn_mlmc_execute_0 = int(os.getenv["ppn_mlmc_execute_0"], 1)
```

https://docs.python.org/3/library/os.html#os.getenv

Reply:
Thanks, I will test it!